### PR TITLE
Hotfix/statistics polling disable

### PR DESCRIPTION
**Pull request by tomereli**
Add a temporary workaround for #527 by disabling the statistics_polling_task

### CODE REVIEW

File: controller/config/beerocks_controller.conf.in [hunk 1]
```diff
@@ -38,6 +38,9 @@ load_monitor_on_vaps=1
 #   health check feature:
 load_health_check=0
 
+#   statistics polling feature
+load_statistics_polling=1
```
Comment (morantr) [inline controller/config/beerocks_controller.conf.in]: We already have a configuration to enable/disable the statistics polling task: `load_diagnostics_measurements`.
So this commit is redundant.

File: controller/src/beerocks/master/son_master_thread.cpp [hunk 2]
```diff
@@ -77,9 +77,11 @@ bool master_thread::init()
...
     set_select_timeout(SOCKETS_SELECT_TIMEOUT_MSEC);
 
-    auto new_statistics_polling_task =
-        std::make_shared<statistics_polling_task>(database, cmdu_tx, tasks);
-    tasks.add_task(new_statistics_polling_task);
+    if (database.settings_statistics_polling()) {
+        auto new_statistics_polling_task =
+            std::make_shared<statistics_polling_task>(database, cmdu_tx, tasks);
+        tasks.add_task(new_statistics_polling_task);
+    }
```
Comment (morantr) [inline controller/src/beerocks/master/son_master_thread.cpp]: I guess it is ok, but basically the task is checking the configuration internally and shut down if it is not enabled.

**Review by morantr — CHANGES_REQUESTED** [review]

File: controller/config/beerocks_controller.conf.in (hunk 1 above)
Comment (tomereli) [reply controller/config/beerocks_controller.conf.in]: ok

File: controller/src/beerocks/master/son_master_thread.cpp (hunk 2 above)
Comment (tomereli) [reply controller/src/beerocks/master/son_master_thread.cpp]: not needed, removed

**Review by arnout — APPROVED** [review]
Make sure that #527 stays open because the bug is still there of course.

**Review by morantr — APPROVED** [review]

**Pull request merged**